package.json

```diff
@@ -124,7 +127,7 @@
     "test": "gulp test",
     "wd": "gulp webdriver-standalone",
     "e2e": "gulp sauce-test",
-    "lint": "gulp lint",
+    "lint": "eslint \"app/**!(*libs)/*.js\" || true",
```

Comment:
Without any issue on gulp-eslint, we don't need to make a call to the CLI tool anymore. Replace this command by 'gulp lint' for consistency.